src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private static final int MAX_RANGE = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int indexOfFigure = random.nextInt(NUMBER_OF_FIGURES);
+        String randomColor = colorSupplier.getRandomColor();
+        int randomParameterA = random.nextInt(MAX_RANGE);
+        int randomParameterB = random.nextInt(MAX_RANGE);
+        int randomParameterC = random.nextInt(MAX_RANGE);
+
+        switch (indexOfFigure) {
+            case 0:
+                return new Square(randomParameterA, randomColor);
```

Comment:
```suggestion
        int indexOfFigure = random.nextInt(NUMBER_OF_FIGURES);

        switch (indexOfFigure) {
            case 0:
                return getRandomSquare();
```
lets add private methods to get each random figure